web/app/scripts/filterbar/numeric-range.html

```diff
@@ -19,20 +19,23 @@
             <div class="input-group">
                 <input type="number"
                        name="minimum"
-                       placeholder="Minimum"
                        class="form-control"
+                       placeholder="Min"
                        min="{{ data.minimum }}"
                        max="{{ data.maximum }}"
                        ng-model="filter.min"
-                       ng-change="updateFilter(label, filter)">
+                       ng-change="updateFilter(label, filter)"
+                       style="width:80px">
```

Comment:
Ah, yeah, looks like that one also snuck in. Yes, if you don't mind, move that one out too.